website/docs/configuration/functions/merge.html.md

```diff
@@ -13,19 +14,33 @@ description: |-
 earlier, see
 [0.11 Configuration Language: Interpolation Syntax](../../configuration-0-11/interpolation.html).
 
-`merge` takes an arbitrary number of maps and returns a single map that
-contains a merged set of elements from all of the maps.
+`merge` takes an arbitrary number of maps or objects, and returns a single map
+pr object that contains a merged set of elements from all arguments. 
```

Comment:
Howdy,  I was just passing by. Sorry if this is a duplicate:

Watch out `pr object` I think you meant `or object`

Reply:
Thanks, totally missed that